config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
+
+  get '/home', to: 'tweets#home'
+  root to: 'tweets#home'
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   devise_for :users
```

Comment:
I notice you have a UsersController, but the only routes you declared for users is within this line. You may wish to run `rake routes` and inspect the controllers those routes are going to.